aclogview/Form1.cs

```diff
@@ -780,54 +766,78 @@ class ListViewItemComparer : IComparer<ListViewItem> {
                     }
                 }
             }
-        }
+        }
+
+
+        private void treeView_ParsedData_AfterSelect(object sender, TreeViewEventArgs e)
+        {
+            updateText();
+        }
+
+
+        private void menuItem_Open_Click(object sender, EventArgs e)
+        {
+            OpenFileDialog openFile = new OpenFileDialog();
+            openFile.AddExtension = true;
+            openFile.Filter = "Packet Captures (*.pcap;*.pcapng)|*.pcap;*.pcapng|All Files (*.*)|*.*";
 
-        private void menuItem_About_Click(object sender, EventArgs e) {
-            MessageBox.Show("aclogview\n\nA program to view and parse Asheron's Call 1 packet capture files (.pcap) generated by aclog.\n\nFor more info and source code, see https://github.com/tfarley/aclogview", "About");
-        }
+            if (openFile.ShowDialog() != DialogResult.OK)
+                return;
 
-        private void treeView_ParsedData_AfterSelect(object sender, TreeViewEventArgs e) {
-            updateText();
+            loadPcap(openFile.FileName);
+        }
+
+
+        private void mnuItem_EditNextHighlightedRow_Click(object sender, EventArgs e)
+        {
+            for (int i = listView_Packets.TopItem.Index + 1; i < listView_Packets.Items.Count; i++)
```

Comment:
If you add a quick null-check here on TopItem, you can prevent the app from crashing when no packets are loaded and you hit F3.